packages/astro/test/astro-styles-ssr.test.js

```diff
@@ -121,6 +121,9 @@ StylesSSR('Astro scoped styles', async ({ runtime }) => {
 
   assert.match(el1.attr('class'), `blue ${scopedClass}`);
   assert.match(el2.attr('class'), `visible ${scopedClass}`);
+
+  const { contents: css } = await runtime.load('/_astro/src/components/Astro.astro.css');
+  assert.match(cssMinify(css.toString()), `.blue.${scopedClass}{color:powderblue}.color\\:blue.${scopedClass}{color:powderblue}.visible.${scopedClass}{display:block}`);
```

Comment:
This test’s purpose isn’t quite testing the scoped transforms; it’s for testing CSS delivery, so I would suggest a better place for this is `astro-scoped-styles.test.js` instead.

Reply:
Oh, but on second thought, it would be good to ensure that the colon in the HTML is left as-is. OK; keep this, but would you mind also adding an additional test to `astro-scoped-styles.test.js` as well? You can add as many cases to test there as you’d like.

Reply:
Ah yep I missed that test file sorry! Makes much more sense 👍 I will update that one instead